index.bs

```diff
@@ -703,6 +703,17 @@ agent MUST run the following steps:
 To <dfn>initialize WebTransport over HTTP</dfn>, given a {{WebTransport}} object
 <var>transport</var>, a [=URL record=] |url|, and a boolean |dedicated|, run these steps.
 
+1. Let |client| be |transport|'s [=relevant settings object=].
+1. Let |request| be a new [=request=] whose [=request/URL=] is |url|, [=request/client=] is
+   |client|, [=request/policy container=] is a clone of |client|'s
+   [=environment settings object/policy container=], [=request/destination=] is an empty string,
+   and [=request/origin=] is |client|'s [=environment settings object/origin=].
+1. If [=should request be blocked by Content Security Policy?=] with |request| returns
```

Comment:
How do I follow this link?  I want to review that process to see if things match.

Reply:
@martinthomson in OP there's a link called "Preview" and a link called "Diff". They will have the links.